moodle/Dockerfile

```diff
@@ -5,7 +5,7 @@ ARG LOCAL_CODECHECKER_VERSION=v5.0.5
 ARG LOCAL_MOODLECHECK_VERSION=v1.3.2
 ARG TOOL_PLUGINSKEL_VERSION=1.6.0
 ARG TOOL_CERTIFICATE_VERSION=v4.3.4
-ARG LOCAL_RAISECLI_VERSION=2023103100
+ARG LOCAL_RAISECLI_VERSION=2024050900-prerelease
```

Comment:
Whoops -- I think you accidentally changed the version tag of the `local_raisecli` plugin instead of `local_raise`.

Reply:
Sure did. Names blended in there - fixed.